src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -84,7 +86,9 @@ public DocumentBuildContext(string buildOutputFolder, IEnumerable<FileAndType> a
 
         public ImmutableArray<string> ExternalReferencePackages { get; }
 
-        public ImmutableArray<string> XRefMapUrls { get; }
+        private ImmutableArray<string> XRefMapUrls { get; }
```

Comment:
>XRefMapUrls [](start = 39, length = 11)

put the private fields separate with publics, like the location of _tableOfContents.